Vizzuality/half-earth-v3
# Add text for no prioritization countries

## Text description for no prioritization countries
## Description
![image](https://user-images.githubusercontent.com/9701591/94438481-74722e80-019f-11eb-9f2a-de8c9c0db9b3.png)
### Testing instructions
Go to a country national report cards sidebar. Check the description of that section
### Feature relevant tickets
https://half-earth-map.atlassian.net/browse/HE-94
